addons/sale_timesheet/models/project.py

```diff
@@ -481,6 +481,10 @@ def merge_profitability_data(a, b):
         )
         return profitability_items
 
+    def _get_domain_aal_with_no_move_line(self):
+        # we add the tuple 'project_id = False' in the domain to remove the timesheets from the search.
+        return super()._get_domain_aal_with_no_move_line() + [('project_id', '=', False)]
```

Comment:
We could use `expression.AND` function to be sure the domain will always valid. 🤷🏼‍♂️ 

```suggestion
        return expression.AND([
            super()._get_domain_aal_with_no_move_line(),
            [('project_id', '=', False)],
        ])
```